test/ios/findProject.spec.js

```diff
@@ -12,20 +12,41 @@ describe('ios::findProject', () => {
 
   it('should return path to xcodeproj if found', () => {
     const userConfig = {};
-    const folder = path.join('testDir', 'flat');
 
-    expect(findProject(folder)).to.contain('.xcodeproj');
+    mockFs(projects.flat);
+
+    expect(findProject('')).to.contain('.xcodeproj');
   });
 
-  it('should ignore xcodeproj from examples', () => {
+  it('should ignore xcodeproj from example folders', () => {
     const userConfig = {};
-    const folder = path.join('testDir', 'withExamples');
 
-    expect(findProject(folder)).to.not.contain('Examples');
+    mockFs({
+      examples: projects.flat,
+      Examples: projects.flat,
+      App: projects.flat,
+    });
+
+    expect(findProject('').toLowerCase()).to.not.contain('examples');
   });
 
+  it('should ignore xcodeproj from test folders at any level', () => {
+    const userConfig = {};
+
+    mockFs({
+      test: projects.flat,
+      IntegrationTests: projects.flat,
+      tests: projects.flat,
+      app: {
+        tests: projects.flat,
+        src: projects.flat,
+      },
+    });
+
+    expect(findProject('').toLowerCase()).to.not.contain('tests');
+  });
 
-  after(() => {
+  afterEach(() => {
```

Comment:
I'd rather leave that for another PR since we have it that way in 99% files, but if you want, I can change that, let me know.